bindings/go/runtime/identity_match_test.go

```diff
@@ -204,3 +204,124 @@ func TestIdentity_Match(t *testing.T) {
 		})
 	}
 }
+
+func TestIdentitySubset(t *testing.T) {
+	type args struct {
+		b runtime.Identity
+		a runtime.Identity
```

Comment:
It makes the test easier to read if you refactor the variable names to what they represent. `a` should be `sub` and `b` should be `base`, right?